modules/unsupported/ysld/src/main/java/org/geotools/ysld/Tuple.java

```diff
@@ -93,33 +118,50 @@ public Tuple parse(List<?> seq) throws IllegalArgumentException {
         }
         return this;
     }
-    
+
+    /**
+     * Parse an object to this Tuple
+     * @param obj
+     * @return
```

Comment:
Is this going to happen a lot? If so just don't have a `@return` if you got noting to say.